src/components/News/index.js

```diff
@@ -19,7 +19,10 @@ const News = props => {
       <ul className='News'>
         {pages[activePage].map((news, index) => (
           <li className='News__item' key={index}>
-            <strong>{news.intro}:</strong> {news.description}
+            <a href={news.link}>
```

Comment:
It seems like some news might not have a link, which is totally fair, but then we should not render a link element at all:

```jsx
<li className='News__item' key={index}>
  {news.link ? (
    <Link to={news.link}><strong>{news.intro}:</strong></Link>
  ) : (
    <strong>{news.intro}:</strong>
  )}{' '}{news.description}
</li>
```